src/main/java/org/opensearch/sdk/ActionExtension.java

```diff
@@ -189,3 +190,5 @@ default Collection<RequestValidators.RequestValidator<IndicesAliasesRequest>> in
     }
 
 }
+
+    
```

Comment:
The little red symbol here tells me this file doesn't end with the proper end of file marker.   And sure enough, it's failing the spotless check in CI.   Running `./gradlew spotlessApply` will fix this.

Reply:
Actually, you shouldn't be changing this file at all!